ENIGMAsystem/SHELL/Universal_System/Extensions/Paths/path_functions.cpp

```diff
@@ -1,6 +1,6 @@
 /********************************************************************************\
 **                                                                              **
-**  Copyright (C) 2011 Harijs Gr�nbergs                                         **
+**  Copyright (C) 2011 Harijs Gr�nbergs                                         **
```

Comment:
dafux this symbol

Reply:
He apparently uses a unicode symbol for the i in his name to have an accent. VS Code must have done this... it probably opened the file using a Windows code page and failed to detect UTF-8... piece of shit....

Reply:
Reverted: bb7ee86555ac65ab0d5ad69dbdaa8d19c947d47f

Reply:
@fundies It's still telling me that it changed. I tried copying it in Notepad++ too and it turns to a square. I don't understand...